tinygrad/runtime/ops_python.py

```diff
@@ -73,7 +73,7 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
                 _store(m, ox*4 + oy*dtp[0].shape[1]*4 + j, v)
           elif dtp[2].sz > 1:
             for j,val in enumerate(inp[2]):
-              for m,o,v in zip(inp[0], inp[1], val): _store(m, o+j, v)
+              for m,o,v in zip(inp[0], inp[1], val if type(val) is list else inp[2]): _store(m, o+j, v)
```

Comment:
Because inp[2] can be just a single float4 vector, which is basically just the same list of values that gets handled in the next elif statement, just labeled with different datatype. Here that case gets handled.

Reply:
See what davidgonmar said about it in discord in #general